kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -182,6 +182,7 @@ func NewJobSubmitCommand(cmdFactory cmdutil.Factory, streams genericclioptions.I
 	cmd.Flags().StringToStringVar(&options.headNodeSelectors, "head-node-selectors", nil, "Node selectors to apply to the head pod in the cluster (e.g. --head-node-selectors topology.kubernetes.io/zone=us-east-1c)")
 	cmd.Flags().StringToStringVar(&options.workerNodeSelectors, "worker-node-selectors", nil, "Node selectors to apply to all worker pods in the cluster (e.g. --worker-node-selectors topology.kubernetes.io/zone=us-east-1c)")
 	cmd.Flags().Int32Var(&options.ttlSecondsAfterFinished, "ttl-seconds-after-finished", 0, "TTL seconds after finished.")
+	cmd.Flags().Float64Var(&options.clusterTimeout, "cluster-timeout", 3600.0, "Timeout in seconds for waiting for the RayCluster to be ready")
```

Comment:
Agree wit the other comment that 120s was too short, but I think 1 hour is too long, can we do something like 10m?